docs/core/install/linux-ubuntu.md

```diff
@@ -21,7 +21,6 @@ The following table is a list of currently supported .NET releases and the versi
 | Ubuntu                 | .NET       |
 |------------------------|------------|
 | [22.04 (LTS)](#2204)   | 6+         |
-| [21.10](#2110)         | 3.1, 6     |
```

Comment:
This table should remove 3.1 since it's no longer supported.

Reply:
What do you mean? The whole table should be removed? 3.1 doesn't go EOL until December.

Reply:
hahha No I mixed up my dates.